eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -0,0 +1,5 @@
+<UsageData>
+  <IgnorePatterns>
+    <UsagePattern IdentityGlob="*/*" />
```

Comment:
For me to better understand, what is this for?

Reply:
Here is the background on the Prebuilt Baseline - https://github.com/dotnet/source-build/blob/main/Documentation/planning/arcade-powered-source-build/onboarding/local-onboarding.md#engsourcebuildprebuiltbaselinexml